backend/src/routes/audit.js

```diff
@@ -211,6 +211,41 @@ module.exports = function(app, io) {
         .catch(err => Response.Internal(res, err))
     });
 
+    // Add findings to audit
+    app.post("/api/audits/:auditId/Allfindings", acl.hasPermission('audits:update'), async function(req, res) {
```

Comment:
It would be better to update the existing route that add a single finding and replace the body requirement from a single object to an array so that it can handle both cases. I would also take advantage of the parameters validation which is not present in this function. 
The `Audit.createFinding` function should also be modified accordingly to update database with one call